lib/customer.rb

```diff
@@ -0,0 +1,40 @@
+require_relative 'menu'
+
+class Customer
+
+  include Menu
+
+  attr_reader :order_array
+
+  def see_menu
+    @menu
+  end
+
+  def choice_validation(item)
+    @menu.has_key?(item) ? :available : :unavailable
```

Comment:
Style/DeprecatedHashMethods: `Hash#has_key?` is deprecated in favor of `Hash#key?`.